app/Jobs/SendNewModuleEmail.php

```diff
@@ -36,7 +36,7 @@ public function handle()
     {
         $mail = $this->user;
 
-        Mail::to((env('RECIEVE_EMAIL')))->send(new GenericEmail($mail));
+        Mail::to((env('RECIEVE_EMAIL')))->send(new NewModuleAvailable($mail));
```

Comment:
Should send to student email if on production ( one of the other mail senders has this for students )